packages/fxa-auth-server/lib/oauthdb/index.js

```diff
@@ -19,9 +19,13 @@
  */
 
 const createBackendServiceAPI = require('../backendService');
+const { OAUTH_SCOPE_SESSION_TOKEN } = require('../constants');
 const { mapOAuthError, makeAssertionJWT } = require('./utils');
+const ScopeSet = require('../../../fxa-shared').oauth.scopes;
+const P = require('../promise');
+const random = require('../crypto/random');
 
-module.exports = (log, config) => {
+module.exports = (log, config, db) => {
```

Comment:
TBH I'm not super thrilled that we have to pass in a reference to `db` here, and I wonder if the graph of dependencies would be cleaner if we moved the "create a sessiontoken" logic into the route, in a similar place to where we currently do the "create a placeholder device revice" part.

But I don't feel strongly about it and would not block landing just for that.